diff --git a/js/3rdparty/webodf/editor/MemberListView.js b/js/3rdparty/webodf/editor/MemberListView.js
--- a/js/3rdparty/webodf/editor/MemberListView.js
+++ b/js/3rdparty/webodf/editor/MemberListView.js
@@ -116,6 +116,7 @@
                 //avatar.getCaret().hideHandle();
             };
             avatarDiv.onclick = function () {
+                documentsMain.onNickChange(memberId, fullnameNode);
                 var caret = editorSession.sessionView.getCaret(memberId);
                 if (caret) {
                     //caret.toggleHandleVisibility();
